Packs/InfobloxBloxOne/Integrations/InfobloxBloxOneThreatDefense/InfobloxBloxOneThreatDefense.py

```diff
@@ -0,0 +1,190 @@
+from CommonServerPython import *
+from CommonServerUserPython import *
+import demistomock as demisto
+
+
+class BloxOneTDClient(BaseClient):
+    def __init__(self, api_key, verify=True, proxy=False):
+        super(BloxOneTDClient, self).__init__(
+            headers={'Authorization': f'Token {api_key}'},
+            base_url='https://csp.infoblox.com',
+            verify=verify,
+            proxy=proxy
+        )
+
+    def dossier_source_list(self) -> list[str]:
+        url_suffix = '/tide/api/services/intel/lookup/sources'
+        res = self._http_request('GET', url_suffix=url_suffix)
+        return [source for source, enabled in res.items() if enabled]
+
+    def lookalike_domain_list(self, *, user_filter: Optional[str] = None,
+                              target_domain: Optional[str] = None, detected_at: Optional[str] = None,
+                              limit: int = 50, offset: Optional[int] = None) -> List[Dict]:
+        url_suffix = '/api/tdlad/v1/lookalike_domains'
+        filter_params = {key: val for key, val in [('_limit', limit), ('_offset', offset)] if val}
+
+        if user_filter:
+            _filter = user_filter
+        elif target_domain:
+            _filter = f'target_domain=="{target_domain}"'
+        else: # detected_at != None
+            _filter = f'detected_at>="{detected_at}"'
```

Comment:
Can't the filter be used with more than one filter value?
If not, please mention it in the description.